cross/node/digests

```diff
@@ -1,3 +1 @@
-node-v0.12.0.tar.gz SHA1 d08810034d170c19759cb38bfc9442ab6b0ebc7a
-node-v0.12.0.tar.gz SHA256 9700e23af4e9b3643af48cef5f2ad20a1331ff531a12154eef2bfb0bb1682e32
-node-v0.12.0.tar.gz MD5 62c8d9c74c8f68193f60e4cba020eb48
+node-v0.12.2.tar.gz SHA256 ac7e78ade93e633e7ed628532bb8e650caba0c9c33af33581957f3382e2a772d
```

Comment:
I didn't know the 'make digests' buildstep and one seemed good enough to me, I didn't understand why three hashes are needed